cc-pk-service/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "cc-pk-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "tsc && node dist/app.js",
+    "dev": "nodemon src/app.ts",
+    "check": "eslint . --ext .ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/express": "^4.17.14",
+    "@types/readline-sync": "^1.4.4",
```

Comment:
is the readline-sync -types used somewhere?

Reply:
I used it before when the program asked for the username and password from the command line, but forgot to remove it after those variables were moved to .env -file. It should be removed now